src/backend/looped/self/wanted_level.cpp

```diff
@@ -1,11 +1,13 @@
 #include "backend/command.hpp"
-#include "backend/looped_command.hpp"
 #include "backend/looped/looped.hpp"
+#include "backend/looped_command.hpp"
+#include "natives.hpp"
 #include "pointers.hpp"
```

Comment:
This include is not required.

Reply:
Removed includes for looped (I assume that's what you meant) and natives since we're not using either.